### PR TITLE
TOOL-317

### PR DESCRIPTION
**Pull request by richard-bogdan-bitrise**
Added test case for a project with androidx dependencies. Fixed support lib regex.

### CODE REVIEW

File: bitrise.yml [hunk 1]
```diff
@@ -240,6 +240,48 @@ workflows:
     - path::./:
         title: Step test
 
+  test_support_library:
+    envs:
+    - SAMPLE_APP_URL: https://github.com/godrei/JetPackTest.git
```
Comment (koral--) [inline bitrise.yml]: Samples should be located in bitrise-io organization.
BTW all other sample URLs in this file points to bitrise-samples org which is deprecated and contains no repos now (there are redirects to bitrise-io).

**Comment by BirmacherAkos** [comment]
It was no your task, but as maintenance please fix the next:

Please update the Bitrise.yml:
- Update the [format version](https://github.com/bitrise-steplib/steps-install-missing-android-tools/blob/e8a5ca6dd6567a8fd806393130f388aa05c33437/bitrise.yml#L1) to the newest (8)
- Our new release flow does not use the [STEP_VERSION](https://github.com/bitrise-steplib/steps-install-missing-android-tools/blob/e8a5ca6dd6567a8fd806393130f388aa05c33437/bitrise.yml#L6) env anymore, it depends on the tag now. Please remove it.
- Please update the [sample-apps-android-sdk22](https://github.com/bitrise-steplib/steps-install-missing-android-tools/blob/e8a5ca6dd6567a8fd806393130f388aa05c33437/bitrise.yml#L9) sample's Github org to `bitrise-io`. We moved all the repos under to `bitrise-io` or `bitrise-steplib` orgs months ago. The `bitrise-sample` ones has been moved to `bitrise-io`
- Do the org update for the other samples as well in the Bitrise.yml.
- Please remove the [create-release](https://github.com/bitrise-steplib/steps-install-missing-android-tools/blob/e8a5ca6dd6567a8fd806393130f388aa05c33437/bitrise.yml#L410-L422) workflow. The new release flow does not use it.
- Please remove the [share-this-step](https://github.com/bitrise-steplib/steps-install-missing-android-tools/blob/e8a5ca6dd6567a8fd806393130f388aa05c33437/bitrise.yml#L435-L473) workflow. The new release flow does not use it.
- Please add a before_run to the ci workflow:
```
before_run:
- audit-this-step
```

File: bitrise.yml (hunk 1 above)
Comment (richard-bogdan-bitrise) [reply bitrise.yml]: Fixed.

**Comment by richard-bogdan-bitrise** [comment]
Fixed review comments.

**Pull request merged**